doc/tutorials/run_kata_containers.rst

```diff
@@ -58,41 +59,88 @@ Install Kata Containers
 ***********************
 
 Kata Containers provide a variety of installation methods, this guide uses
-:command:`kata-manager` to automate the Kata Containers installation procedure.
+`kata-deploy <https://github.com/kata-containers/packaging/tree/master/kata-deploy>`_
+to automate the Kata Containers installation procedure.
 
-#. Install Kata Containers packages:
+#. Install Kata Containers:
 
    .. code-block:: none
 
-      $ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/tests/master/cmd/kata-manager/kata-manager.sh) install-packages"
+      $ sudo docker run -v /opt/kata:/opt/kata -v /var/run/dbus:/var/run/dbus -v /run/systemd:/run/systemd -v /etc/docker:/etc/docker -it katadocker/kata-deploy kata-deploy-docker install
 
-#. Add the following settings to :file:`/etc/docker/daemon.json` to configure
-   Docker to use Kata Containers by default. You may need to create the
-   file if it doesn't exist.
+#. Install the ``acrnctl`` tool:
 
    .. code-block:: none
 
+      $ cd /home/acrn/work/acrn-hypervisor
+      $ sudo cp build/misc/tools/acrnctl /usr/bin/
+
+   .. note:: This assumes you have built ACRN on this machine following the
+      instructions in the :ref:`rt_industry_ubuntu_setup`.
+
+#. Modify the :ref:`daemon.json` file in order to:
+
+   #. Add a ``kata-acrn`` runtime (``runtimes`` section).
+
+      .. note:: In order to run Kata with ACRN, the container stack must provide
+         block-based storage, such as :file:`device-mapper`. Since Docker may be
+         configured to use :file:`overlay2` storage driver, the above
+         configuration also instructs Docker to use :file:`devive-mapper`
+         storage driver.
+
+   #. Use the ``device-mapper`` storage driver.
+
+   #. Make Docker use Kata Containers by default.
+
+   These changes are highlighted below.
+
+   .. code-block:: none
+      :emphasize-lines: 2,3,21-24
+      :name: daemon.json
+      :caption: /etc/docker/daemon.json
+
       {
         "storage-driver": "devicemapper",
-        "default-runtime": "kata-runtime",
+        "default-runtime": "kata-acrn",
```

Comment:
why rename the runtime? If kata generates runtime as kata-runtime it is better to use it that way.